README.md

```diff
@@ -142,15 +146,19 @@ If Swagger format is configured, [Swagger UI](https://swagger.io/tools/swagger-u
 You can add this configurations to your `config/test.ex`
 
 -   information_source: the module with doc information
--   output_file: a custom name to the output file
--   doc_format: ApiBlueprint or Swagger formats
+-   output: a custom name to the output file
+-   format: ApiBlueprint or Swagger formats
 -   env_var: a custom name to the env to active XCribe.Formatter
+-   json_library: The library to be used for json decode/encode
 
 Example
 
 ```elixir
-config :xcribe, :information_source, YourAppWeb.Information
-config :xcribe, :output_file, "API-DOCUMENTATION.apib"
-config :xcribe, :doc_format, :swagger # or :api_blueprint
-config :xcribe, :env_var, "DOC_API"
+config :xcribe, :configuration, [
```

Comment:
Any specific reason to add this `:configuration` key? Instead of just adding directly to the `:xcribe` config?

Reply:
This is a vestige of an idea that I had but it's not needed now. Tks  :+1: